includes/Withdraw/Manager.php

```diff
@@ -266,6 +267,22 @@ public function get_status_code( $status ) {
         return null;
     }
 
+    /**
+     * Get withdraw details by method and user
+     *
+     * @param  string $method
+     * @param  int    $user_id
+     *
+     * @return integer
+     */
+    public function get_formatted_details( $method, $user_id ) {
+        $vendor = dokan()->vendor->get( $user_id );
+
+        if ( isset( $vendor->get_payment_profiles()[$method] ) ) {
+            return [ $method => $vendor->get_payment_profiles()[$method] ];
```

Comment:
Same here - 
We should give space before starting and ending a variable according to phpcs
`[ $method ]`